tools/commands-cordova.js

```diff
@@ -1443,7 +1443,8 @@ var consumeControlFile = function (
   // set some defaults different from the Phonegap/Cordova defaults
   var additionalConfiguration = {
     'webviewbounce': false,
-    'DisallowOverscroll': true
+    'DisallowOverscroll': true,
+    'deployment-target': "7.0"
```

Comment:
For being compliant to the existent code, it should be `'7.0'` and not `"7.0"`. Am I wrong? :eyes: